sdk/formrecognizer/Azure.AI.FormRecognizer/src/Azure.AI.FormRecognizer.csproj

```diff
@@ -1,11 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Description>This is a template project to demonstrate how to create a package that uses code generation as well as use for testing our build and release pipelines</Description>
-    <AssemblyTitle>Azure SDK Template</AssemblyTitle>
-    <Version>1.0.2-preview.6</Version>
-    <PackageTags>Azure Template</PackageTags>
+    <Description>This is the Microsoft Azure Cognitive Services Form Recognizer client library</Description>
+    <AssemblyTitle>Microsoft Azure.AI.FormRecognizer client library</AssemblyTitle>
+    <Version>1.0.0-preview.1</Version>
```

Comment:
@annelo-msft not sure what the version is going to be... so I went with this for now

Reply:
That looks correct.  We'll confirm before release.